Error on import to test file when using @wordpress/scripts to run tests

**Describe the bug**
When running `yarn test:unit` in a plugin using `@wordpress/scripts` I get errors like this:
`    Cannot find module 'core-js/modules/es.date.to-json' from 'Form.test.js'` It is triggered by the first import in any test files.

I fixed it by adding `@core-js` after reading this: https://github.com/TryGhost/gatsby-starter-ghost/issues/29#issuecomment-478265557


**To reproduce**
Steps to reproduce the behavior:
1. Add a test
2. `yarn test:unit`
3. See error described above
4. See error

**Expected behavior**
This should not be required.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version "@wordpress/scripts@3.3.0"
- Node v10.16.0

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone6]
 - OS: [e.g. iOS8.1]
 - Browser [e.g. stock browser, safari]
 - Version [e.g. 22]

--- a/package.json
+++ b/package.json
@@ -95,7 +95,6 @@
 		"commander": "2.20.0",
 		"concurrently": "3.5.0",
 		"copy-webpack-plugin": "4.5.2",
-		"core-js": "3.0.1",
 		"cross-env": "3.2.4",
 		"cssnano": "4.1.10",
 		"deep-freeze": "0.0.1",

--- a/packages/babel-preset-default/CHANGELOG.md
+++ b/packages/babel-preset-default/CHANGELOG.md
@@ -1,3 +1,10 @@
+## Master
+
+### Bug Fixes
+
+- Configure Babel to target your current version of Node as described in [Jest docs](https://jestjs.io/docs/en/getting-started#using-babel).
+- Added missing [core-js](https://www.npmjs.com/package/core-js) dependency ([#16259](https://github.com/WordPress/gutenberg/pull/16259)).
+
 ## 4.2.0 (2019-05-21)
 
 ### New Features

--- a/packages/babel-preset-default/index.js
+++ b/packages/babel-preset-default/index.js
@@ -18,8 +18,9 @@ module.exports = function( api ) {
 		const opts = {};
 
 		if ( isTestEnv ) {
-			opts.useBuiltIns = 'usage';
-			opts.corejs = 3;
+			opts.targets = {
+				node: 'current',
+			};
 		} else {
 			opts.modules = false;
 			opts.targets = {

--- a/packages/babel-preset-default/package.json
+++ b/packages/babel-preset-default/package.json
@@ -35,7 +35,8 @@
 		"@babel/preset-env": "^7.4.4",
 		"@babel/runtime": "^7.4.4",
 		"@wordpress/babel-plugin-import-jsx-pragma": "file:../babel-plugin-import-jsx-pragma",
-		"@wordpress/browserslist-config": "file:../browserslist-config"
+		"@wordpress/browserslist-config": "file:../browserslist-config",
+		"core-js": "^3.1.4"
 	},
 	"peerDependencies": {
 		"@babel/core": "^7.0.0"

--- a/packages/scripts/CHANGELOG.md
+++ b/packages/scripts/CHANGELOG.md
@@ -4,6 +4,10 @@
 
 - The `build` and `start` commands supports simplified syntax for multiple entry points: `wp-scripts build entry-one.js entry-two.js` ([15982](https://github.com/WordPress/gutenberg/pull/15982)).
 
+### Bug Fix
+
+- Added missing [babel-jest](https://www.npmjs.com/package/babel-jest) dependency ([#16259](https://github.com/WordPress/gutenberg/pull/16259)).
+
 ## 3.3.0 (2019-06-12)
 
 ### New Features

--- a/packages/scripts/package.json
+++ b/packages/scripts/package.json
@@ -37,6 +37,7 @@
 		"@wordpress/eslint-plugin": "file:../eslint-plugin",
 		"@wordpress/jest-preset-default": "file:../jest-preset-default",
 		"@wordpress/npm-package-json-lint-config": "file:../npm-package-json-lint-config",
+		"babel-jest": "^24.7.1",
 		"babel-loader": "^8.0.5",
 		"chalk": "^2.4.1",
 		"check-node-version": "^3.1.1",

